### PR TITLE
[optimization] support 8bit optims from bistandbytes

### LINKED ISSUE #137: Support 8bit optimizers from `bitsandbytes`

### PR DESCRIPTION
**Pull request by sayakpaul**
Fixes #137. 

Run: https://wandb.ai/sayakpaul/finetrainers-ltxv/runs/7kk3xhkh

### CODE REVIEW

File: finetrainers/args.py [hunk 1]
```diff
@@ -79,6 +79,7 @@ class Args:
 
     # Optimizer arguments
     optimizer: str = "adamw"
```
[the remark this answers is not in the document]
Comment (sayakpaul) [reply finetrainers/args.py:81]: Done.

File: finetrainers/args.py [hunk 2]
```diff
@@ -79,6 +79,7 @@ class Args:
...
+    use_8bit_bnb_bnb: bool = False
```
Comment (a-r-r-o-w) [inline finetrainers/args.py:82]: This is incorrect, no?

**Review by a-r-r-o-w — APPROVED** [review]
Good to merge after the fix

**Comment by sayakpaul** [comment]
Working fine with all the changes: https://wandb.ai/sayakpaul/finetrainers-ltxv/runs/kl9swsq8.

I have added a note in the README too.

**Pull request merged**